remote.go

```diff
@@ -1200,7 +1200,7 @@ func (r *Remote) updateLocalReferenceStorage(
 
 			// If the ref exists locally as a branch and force is not specified,
 			// only update if the new ref is an ancestor of the old
-			if old != nil && old.Name().IsBranch() && !force && !spec.IsForceUpdate() {
+			if old != nil && !old.Name().IsTag() && !force && !spec.IsForceUpdate() {
```

Comment:
Please amend the comments above to align with the changes.

Reply:
Done!